docs/data-sources.md

```diff
@@ -110,9 +131,9 @@ zoom   | meters at equator     | arc seconds     | nominal arc degrees minutes s
 
 ## Data updates
 
-Terrain tiles were built during 2016Q2 and 2016Q3 based on available sources at that time. Regular updates are not planned.
+Terrain tiles were built during 2017Q2 based on available sources at that time. Regular updates are not planned.
```

Comment:
Since the endpoints are versioned, we should not 2017Q2 for v2 and keep the note that v1 was built in 2016Q2.